index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Panda_login</title>
+  <link rel="icon" href="images/logo/favicon.ico">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  
+  <div class="container">
+    <div class="logo"><a href="/">
+    <img src="images/logo/logo.svg" width="396px" height="132px"></a>
+    </div>
+ 
+    <div class="login_box">
```

Comment:
❗️ 수정요청
해당 영역은 로그인할때 필요한 데이터들을 받는 부분이니 form으로 묶는 것이 더 시멘틱하고 좋습니다.
또한 form으로 input과 button을 포함하게 되면, 해당 form의 상호작용가능한 자식요소에서 onsubmit 이벤트가
발생시 form이 이를 감지할 수 있고, 유저입장에서 enter 키를 눌렀을 때 onsubmit 이벤트가 제출되므로 UX 적으로도 좋습니다.

```suggestion
    <form class="login_box">
```

```html
// case1: form에 onsubmit 이벤트 명시
<form class="login_box" onsubmit="login">
  <input /> // input에서 enter 키 입력시 onsubmit 이벤트 실행
  <button type="submit">login</button> // 해당 버튼 클릭 시 onsubmit 이벤트 실행
</form>

// case2: submit 버튼에 onclick 이벤트 명시
<form class="login_box">
  <input /> // input에서 enter 키 입력시 button의 onclick 이벤트 실행
  <button type="submit" onclick="login">login</button>
</form>
```